### ISSUE TITLE
Disconnect and connect GitHub account

### ISSUE THREAD

**Issue by driesvints** [laravelio/laravel.io#659]
Users should be able to connect or disconnect their GitHub account after creating their account. Should be a new box on the settings page. Connecting their GitHub account will fill in their GitHub username. 

Explain why we ask to connect their GitHub account (as opposed to let them fill in a text field for their username):

1. We offer social authentication this way
2. We use their GitHub account to verify they're real persons (and not bots)
3. We integrate with GitHub for upcoming features

**Comment by abrardev99** [comment]
Hey Dries, 
I like to work on this issue. Just need to know that no one already working on this feature?

**Comment by driesvints** [comment]
No one has replied so far here so I don't think so.

**Comment by driesvints** [comment]
Yeah we'd still appreciate a PR. In general you don't have to wait for another. Anyone is free to work on any issue with the "help wanted" label.
